### ISSUE TITLE
CocoaPods trunk.cocoapods.org returns 'low level' error

### ISSUE THREAD

**Issue by abower-digimarc** [CocoaPods/trunk.cocoapods.org#458]
Tried to visit the url https://trunk.cocoapods.org/claims/new, but receive only the error below.

"An unhandled lowlevel error occurred. The application logs may have details."

**Comment by abower-digimarc** [comment]
Additionally, we had a developer attempt to request access to our pod/spec/trunk, and we received the e-mail to approve. Clicking the link in the request e-mail to allow that access resulted in the exact same error message shown above.

**Comment by gabrieldonadel** [comment]
Same problem here

**Comment by orta** [comment]
I'm unlikely to be able to make time to do this anytime soon I'm afraid, but this is likely overreach from https://github.com/CocoaPods/trunk.cocoapods.org/commit/71be5440906b6bdfbc0bcc7f8a9fec33367ea0f4

**Comment by pradeep23Edcast** [comment]
Same problem here

**Comment by tonyarnold** [comment]
Yeah, I can't get past this either.

**Comment by tonyarnold** [comment]
Has anyone found a workaround to this issue? I released an update yesterday, and currently there's no way for me (or anyone else on my team) to update the pod spec.

@orta could 71be5440906b6bdfbc0bcc7f8a9fec33367ea0f4 be rolled back?

**Comment by orta** [comment]
No, that's a vulnerability, someone could rollback the commit and just remove the ability for someone to get assigned to a podspec without any owners though

**Comment by orta** [comment]
The `pod trunk register` issue was it bailing on showing the 'congrats' screen - the CLI should have been fine. Switched to simple text rendering instead

![image](https://github.com/CocoaPods/trunk.cocoapods.org/assets/49038/188cf23c-813f-4cec-a269-4c544d324095)

**Comment by tonyarnold** [comment]
Thanks @orta - this is working for me now. I really appreciate the thoroughness, and that you were dealing with a CVE.
